.woke.yml

```diff
@@ -0,0 +1,9 @@
+rules:
+  # Ignore "whitelist" - upstream uses this terminology and it would be
+  # confusing to change in this charm while upstream continues to use it.
+  # Specifically the following annotations:
+  #   - nginx.ingress.kubernetes.io/limit-whitelist
+  #   - nginx.ingress.kubernetes.io/whitelist-source-range
+  # See https://github.com/kubernetes/ingress-nginx/issues/7916 for upstream
+  # issue about this.
+  - name: whitelist
```

Comment:
I think it would be better to ignore the library file (see https://github.com/canonical/indico-operator/blob/main/.woke.yaml for an example) than globally ignoring it in this whole project.

Reply:
Sure, thanks! Fixed.